src/FSharp.Control.TaskSeq.Test/TestUtils.fs

```diff
@@ -95,23 +95,25 @@ type DummyTaskFactory(µsecMin: int64<µs>, µsecMax: int64<µs>) =
     /// <summary>
     /// Creates dummy tasks with a randomized delay and a mutable state,
     /// to ensure we properly test whether processing is done ordered or not.
-    /// Uses the defaults for <paramref name="µsecMin" /> and <paramref name="µsecMax" />
+    /// Uses the defaults for <paramref name="minMs" /> and <paramref name="maxMs" />
```

Comment:
This is an incorrect change. The param refs are clearly named `µsecMin` etc, see above. This should be reversed:
```suggestion
    /// Uses the defaults for <paramref name="µsecMin" /> and <paramref name="µsecMax" />
```